Gems/PhysX/Code/Source/StaticRigidBodyComponent.cpp

```diff
@@ -56,15 +57,11 @@ namespace PhysX
 
     void StaticRigidBodyComponent::GetIncompatibleServices(AZ::ComponentDescriptor::DependencyArrayType& incompatible)
     {
-        // There can be only one StaticRigidBodyComponent per entity
-        incompatible.push_back(AZ_CRC_CE("PhysicsStaticRigidBodyService"));
-        // Cannot have both StaticRigidBodyComponent and RigidBodyComponent
         incompatible.push_back(AZ_CRC_CE("PhysicsRigidBodyService"));
     }
 
-    void StaticRigidBodyComponent::GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent)
+    void StaticRigidBodyComponent::GetDependentServices([[maybe_unused]] AZ::ComponentDescriptor::DependencyArrayType& dependent)
```

Comment:
same comment as before

Reply:
Answered in the first comment of this kind.